pyclam/manifold.py

```diff
@@ -87,7 +87,7 @@ def __str__(self) -> str:
 
     def __repr__(self) -> str:
         if 'repr' not in self.cache:
-            self.cache['repr'] = '-'.join([self.name, ', '.join(map(str, self.argpoints))])
+            self.cache['repr'] = '-'.join([self.name, ', '.join(map(str, sorted(self.argpoints)))])
```

Comment:
are we sure we want to sort the argpoints?

Reply:
I like the visual, while debugging, of having indices increase as I read them from left to right